<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AimingBlockWindow</class>
 <widget class="QWidget" name="AimingBlockWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>177</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AimingBlockWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
border: 2px solid line black;
	background-color: rgb(148, 134, 115);
	font: 14pt &quot;Sans Serif&quot;;
}
QLineEdit
{
border: 2px solid line black;
	
	background-color: rgb(227, 227, 227);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
border: 2px solid line black;
	
	background-color: rgb(255, 170, 0);
}</string>
     </property>
     <property name="text">
      <string>Блок наведения</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>80</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>:/MainControllerQClass/TrainerPictures/Контроллер3.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">QGroupBox
{
border: 1px solid line;
	color: rgb(148, 148, 148);
}


QPushButton
{
background-color: rgb(170, 0, 0);
}

QPushButton:checked
{

 
	background-color: rgb(85, 170, 0);

}

QPushButton:pressed
{

 
	
	background-color: qlineargradient(spread:pad, x1:0.025, y1:0.00568182, x2:1, y2:1, stop:0 rgba(0, 170, 18, 255), stop:1 rgba(255, 255, 255, 255));

}
</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkMoveSlowPID">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox
{
border: 2px solid line black;
	
	
	background-color: rgb(208, 0, 0);
}

QCheckBox::checked
{
border: 2px solid line black;
	
	background-color: rgb(85, 170, 0);
}</string>
        </property>
        <property name="text">
         <string>Медленно</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkMoveFastPID">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox
{
border: 2px solid line black;
	
	
	background-color: rgb(208, 0, 0);
}

QCheckBox::checked
{
border: 2px solid line black;
	
	background-color: rgb(85, 170, 0);
}</string>
        </property>
        <property name="text">
         <string>Быстро</string>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="checkWorkBlock">
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox
{
border: 2px solid line black;
	
	
	background-color: rgb(208, 0, 0);
}

QCheckBox::checked
{
border: 2px solid line black;
	
	background-color: rgb(85, 170, 0);
}</string>
     </property>
     <property name="text">
      <string>Работа</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
